### PR TITLE
Issue/5090 style aztec integration

### PR DESCRIPTION
**Pull request by theck13**
### Fix
Style Aztec integration like Calypso as described in https://github.com/wordpress-mobile/WordPress-Android/issues/5090 and https://github.com/wordpress-mobile/WordPress-Android/issues/5068.

### Test
1. Open Aztec editor in the WordPress app.
2. Open Calypso editor in a web browser.
3. Compare styling.

### CODE REVIEW

File: libs/editor/WordPressEditor/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -1,18 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
+
 <resources>
+
+    <color name="format_bar_background">#f9fbfc</color>
+    <color name="format_bar_button_highlighted_color">@color/wp_blue</color>
+    <color name="format_bar_button_normal_color">@color/wp_grey_lighten_10</color>
```
Comment (khaykov) [inline libs/editor/WordPressEditor/src/main/res/values/colors.xml]: I cant find any usage of this two resources (format_bar_button_xx). Should we remove them?

**Comment by khaykov** [comment]
I noticed that color of selected toolbar button is different then in calypso - Aztec is using wp_blue while calypso is using some dark color. Is it intentional?

**Comment by theck13** [comment]
Yes, that was intentional.  Aztec is using [`wp_blue`](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/5090-style-aztec-integration/libs/editor/WordPressEditor/src/main/res/values/wp_colors.xml#L5) (#0087be) and Calypso is using [`wp_grey_dark`](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/5090-style-aztec-integration/libs/editor/WordPressEditor/src/main/res/values/wp_colors.xml#L10) (#2e4453) for highlighted buttons.  The reason for using a different color than Claypso is to differentiate more clearly between highlighted and unhighlighted buttons since the unhighlighted color is very similar with [`wp_grey_darken_20`](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/5090-style-aztec-integration/libs/editor/WordPressEditor/src/main/res/values/wp_colors.xml#L12) (#4f748e).  Calypso outlines the highlighted button as well which makes it easy to see whereas Aztec does not.  That's might be something we should change in the future or do something more like Google Docs with a different background color for highlighted buttons.

buttons

**Comment by khaykov** [comment]
Awesome, thanks for the explanation, @theck13 ! 
:shipit:

**Pull request merged**